src/client/src/components/SelectableCard/index.tsx

```diff
@@ -136,12 +142,6 @@ const SelectableCard = ({
           >
             {(isPagesSelection && <div>{clickCount}</div>) || (
               <div className={styles.selectedText}>
-                <div>
-                  <FormattedMessage
-                    id="selectableCard.selected"
-                    defaultMessage="Selected"
```

Comment:
is removing the "selected" message ok from the accessibility standpoint? will screen reader knows which card is selected without the message?

Reply:
The screen reader never picked up the message before I removed it. There are a lot of accessibility problems like this throughout the wizard which I will work on in another PR